pkg/tfbridge/schema.go

```diff
@@ -190,6 +189,11 @@ const defaultsKey = "__defaults"
 // marshaled back to assets by MakeTerraformOutputs.
 type AssetTable map[*SchemaInfo]resource.PropertyValue
 
+// SchemaDefaultValueErr catches the situation where pf/schemashim.DefaultValue() is called by ExtractInputsFromOutputs.
+// To avoid a panic, we return this error and log it to Info.
+// See https://github.com/pulumi/pulumi-cloudflare/issues/460
+var SchemaDefaultValueErr = fmt.Errorf("default values not supported")
```

Comment:
You can just say `Internal` also in the comment - not something we expect users to need to link to.